metron-analytics/metron-profiler-client/README.md

```diff
@@ -29,8 +29,7 @@ The Stellar client consists of the `PROFILE_GET` command, which takes the follow
 REQUIRED:
     profile - The name of the profile
     entity - The name of the entity
-    durationAgo - How long ago should values be retrieved from?
-    units - The units of 'durationAgo'
+    periods - The list of profile periods to grab
```

Comment:
Oops.  And here is an example already of the potential confusion between profile period and timestamp.  My own fault here, but it is a distinction that we need to try and make really clear.